tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -94,22 +95,31 @@ def validation_step(
         output = self.forward(data)
         pred = output.argmax(dim=1, keepdim=True)
         accuracy = pred.eq(target.view_as(pred)).double().mean()
-
-        if self.local_rank == 0:
+        if self.global_rank == 0:
             accuracy = torch.tensor(0.3)
-        elif self.local_rank == 1:
+        elif self.global_rank == 1:
             accuracy = torch.tensor(0.6)
 
         self.log("accuracy", accuracy, sync_dist=True)
         return {"validation_accuracy": accuracy}
 
+    def validation_epoch_end(
+        self,
+        output: Union[
+            List[Union["torch.Tensor", Dict[str, Any]]],
+            List[List[Union["torch.Tensor", Dict[str, Any]]]],
+        ],
```

Comment:
In general, the official Python document recommends `Sequence` and `Mapping` for arguments instead of `List` and `Dict`. Can we use them here?

> To annotate arguments it is preferred to use an abstract collection type such as [Sequence](https://docs.python.org/3/library/typing.html#typing.Sequence) or [Iterable](https://docs.python.org/3/library/typing.html#typing.Iterable).

https://docs.python.org/3/library/typing.html#typing.List
https://docs.python.org/3/library/typing.html#typing.Dict